.gitignore

```diff
@@ -5,3 +5,4 @@ phpunit.xml
 /vendor/
 /tests/Fixtures/app/cache/*
 /tests/Fixtures/app/logs/*
+/src/Bridge/Symfony/Bundle/Resources/public
```

Comment:
I don't think we should use the `public` directory itself directly, but rather a subdirectory `public/swagger-ui`.